### PR TITLE
Add  Ubuntu 20.04 LTS

### PR DESCRIPTION
**Pull request by twistedgrim**
Corresponding Issue(s):

https://jira.rax.io/browse/MPCSUPENG-1512

Summary of change(s):

Added Ubuntu 20.04

Reason for Change(s):

Requested addition of ubuntu 20.04

Will the change trigger resource destruction or replacement? If yes, please provide justification:

NA new

Does this update/change involve issues with other external modules? If so, please describe the scenario.

no
If input variables or output variables have changed or has been added, have you updated the README?

readme is updated completely
Do examples need to be updated based on changes?

they were updated
Note to the PR requester about Closing PR's

Please message the person that opened the issue when auto closing it on slack, as well as any other stake holders of deep interest. Only close the issue if you believe that the issue is fully resolved with this PR.
This PR may auto close the issue associated with it. If you feel the issue is not resolved please reopen the issue.

### CODE REVIEW

**Review by bohn002 — APPROVED** [review]

File: main.tf [hunk 1]
```diff
@@ -691,6 +697,7 @@ module "cpu_alarm_high" {
   rackspace_managed        = var.rackspace_managed
   statistic                = "Average"
   threshold                = var.cw_cpu_high_threshold
+  unit                     = "Count"
```
Comment (johnctitus) [inline main.tf:700]: Hate to be nitpicky, but been burned by this before.  Can\have we confirmed the alarm is behaving as expected with this unit in use?  

I only ask, as AWS docs has the official unit as `Percentage` and have seen instances in the past where incorrect units resulted in alarms being stuck in the `INSUFFICIENT_DATA` state.

**Review by johnctitus — COMMENTED** [review]
Code looks good, just would like confirmation on one point

File: main.tf (hunk 1 above)
Comment (twistedgrim) [reply main.tf:700]: It was nothing before just an empty string as default. It needed something as it was failing CI. Ill change it to percent.

**Review by johnctitus — APPROVED** [review]
👍  Thanks for humoring me.

**Review by cwgem — APPROVED** [review]

**Pull request merged**